src/ng/directive/input.js

```diff
@@ -153,13 +155,15 @@ var inputType = {
             }]);
        </script>
        <form name="myForm" ng-controller="DateController as dateCtrl">
-          Pick a date in 2013:
+          <label for="exampleInput">Pick a date in 2013:</label>
```

Comment:
It looks like `exampleInput` is a duplicate `id`. Will that impact focus and AT behavior?